### DISCUSSION TITLE
How to install pkg globally with specific registry?

### DISCUSSION THREAD

**Discussion by zheeeng** [pnpm/pnpm#4738] [Q&A]
I'm not sure what is the alternative for `npm install --registry=https://registry.somewhere.com -g some-pkg`.

Change `npm` to `pnpm` reports `This modules directory was created using the following registries configuration: {"default":"https://registry.npmjs.org/"}. The current configuration is {"default":"https://registry.somewhere.com/"}. To recreate the modules directory using the new settings, run "pnpm install".`.

**Comment by asins** [comment]
I also prompted this issue when removing a global command installed via a private registry.

```bash
# install success
pnpm rm -g @aa/bb --registry=http://registry.xxxx.com/
# remove error
pnpm rm -g @aa/bb
```
